hal/src/rtl872x/ble_hal.cpp

```diff
@@ -229,7 +238,7 @@ class BleEventDispatcher {
     void* allEvtQueue_;
     void* ioEvtQueue_;
     bool started_;
-    static constexpr uint8_t BLE_EVENT_THREAD_PRIORITY = OS_THREAD_PRIORITY_NETWORK;
+    static constexpr uint8_t BLE_EVENT_THREAD_PRIORITY = OS_THREAD_PRIORITY_NETWORK - 1; // IMPORTANT: below network for coexistence to work!
```

Comment:
Any thread managing WiFi would effectively be higher priority than the BLE thread?